app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -268,6 +287,7 @@ public Study createStudy(Study study) {
         study.setActive(true);
         study.setStrictUploadValidationEnabled(true);
         study.setEmailVerificationEnabled(true);
+        study.setEmailSignInEnabled(false);
```

Comment:
Added to StudyServiceTest alongside tests for the other flags through create/update with different roles. (crudStudy test).